### PR TITLE
Symfony 3.0 support

### PR DESCRIPTION
**Pull request by whyte624**
https://github.com/helthe/Turbolinks/issues/7

### CODE REVIEW

**Comment by whyte624** [comment]
Is it possible to merge this pull request?

**Comment by oliverklee** [comment]
@whyte624 The Travis build failed, but it does not seem to be related to your code AFAICT. Could you please change something in your PR (e.g., squash the two commits into one) and repush (using --force) to trigger another build?

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```
Comment (oliverklee) [inline .travis.yml]: This does not seem to be related to the Symfony 3.0 support. I propose to move it to a separate PR so we have "one PR per issue".

**Comment by carlalexander** [comment]
Sorry, I haven't had a chance to look at this. Can you remove the PHP 7 commit as @oliverklee suggested and see if the tests pass?

**Comment by carlalexander** [comment]
Actually, to be fair, this was related to the 5.3.3 test. I just removed it. If you just rebase your changes, I'll merge them in.

**Comment by whyte624** [comment]
Done!

**Comment by carlalexander** [comment]
I'll tag a new version in a bit

**Comment by whyte624** [comment]
Tag, please!

**Comment by carlalexander** [comment]
So sorry! Thought I did. It's there now

**Comment by whyte624** [comment]
Great, thanks!
